runtime-lint/index.js

```diff
@@ -0,0 +1,65 @@
+// true/false in attributes
+block('*').def()(function() {
+  var ret = applyNext();
+
+  var ctx = this.ctx;
+  var attrs = ctx.attrs;
+  if (attrs) {
+    Object.keys(attrs).forEach(function(key) {
+      if (typeof attrs[key] === 'boolean') {
+        console.warn(
+          '\nBEM-XJST WARNING: boolean attribute value:',
+          attrs[key],
+          'in BEMJSON:',
```

Comment:
Suddenly specification draft for Console API has multiple arguments: https://console.spec.whatwg.org/#warn-data
But who knows when it becomes alive.